Core/MappingResolution/UnmappedItemException.cs

```diff
@@ -24,7 +24,9 @@ namespace Remotion.Linq.SqlBackend.MappingResolution
   /// The <see cref="UnmappedItemException"/> is thrown if the <see cref="IMappingResolver"/> implementation 
   /// is unable to associate an part of the query model with the mapping.
   /// </summary>
+#if NETFRAMEWORK || NET6_0 || NET7_0
```

Comment:
There is also the variable `NET8_0_OR_GREATER` which I would prefer instead of ORing together the individual variables

```suggestion
#if !NET8_0_OR_GREATER
```